modules/gdnative/gdnative.cpp

```diff
@@ -858,12 +868,56 @@ bool GDNativeInstance::get(const StringName &p_name, Variant &r_ret) const {
 		r_ret = *(Variant *)&value;
 		return true;
 	}
+
+	Map<StringName, GDNativeScriptData::Method>::Element *E = script->script_data->methods.find("_get");
+	if (E) {
+		Variant name = p_name;
+		const Variant *args[1] = { &name };
+
+		godot_variant result = E->get().method.method((godot_object *)owner, E->get().method.method_data, userdata, 1, (godot_variant**) args);
+		r_ret = *(Variant *)&result;
+		return true;
+	}
+
 	return false;
 }
 
 void GDNativeInstance::get_property_list(List<PropertyInfo> *p_properties) const {
 	script->get_script_property_list(p_properties);
 	// TODO: dynamic properties
+
+	Map<StringName, GDNativeScriptData::Method>::Element *E =  script->script_data->methods.find("_get_property_list");
+	if (E) {
+		godot_variant result = E->get().method.method((godot_object *)owner, E->get().method.method_data, userdata, 0, NULL);
+		Variant ret = *(Variant *)&result;
```

Comment:
I see you use that weird C-style casting everywere.

Why not put it in some outside function like `cast_to_Variant(godot_variant gv)` (that is probably bad name) and inside it do that casting, if you use it so often? (CC @karroffel, maybe he should make it)

Also it's better to use `reinterpret_cast` - it's easier to find in code, will not break const correctness, etc.